sdcm/scan_operation_thread.py

```diff
@@ -460,6 +474,9 @@ def execute_query(self, session, cmd: str,
                                   | FullPartitionScanReversedOrderEvent]) -> None:
         self.log.debug('Will run command %s', cmd)
         validate_mapreduce_service_requests_start_time = time.time()
+        session.cluster.default_retry_policy = ExponentialBackoffRetryPolicy(**self._exp_backoff_retry_policy_params)
+        session.default_timeout = self._session_execution_timeout
```

Comment:
is there a difference between `self._request_default_timeout` and `self._request_default_timeout`? maybe it could be reused?